NOTES.ANDROID

```diff
@@ -0,0 +1,67 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Besides basic tools like perl and make you'll need to download Android
+ NDK. It's avialable for Linux, MacOS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that MacOS X
+ wouldn't work. And as for Windows it's unclear which "shell" would work
+ [best], MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support any recent enough.
```

Comment:
"any recent enough" doesn't sounds quite right...  do you mean "anything that's recent enough"?